src/test/java/org/assertj/core/internal/maps/Maps_assertAllSatisfyingConsumer_Test.java

```diff
@@ -50,36 +54,47 @@ public void should_pass_if_all_entries_satisfy_the_given_requirements() {
 
   @Test
   public void should_pass_if_actual_map_is_empty() {
+    // GIVEN
     greatPlayers.clear();
-    maps.assertAllSatisfy(someInfo(), greatPlayers, (team, player) -> {
-      // whatever, this is never called
-      assertThat(player.getPointsPerGame()).isGreaterThan(200);
+
+    // WHEN THEN
+    maps.assertAllSatisfy(someInfo(), greatPlayers, ($1, $2) -> {
+      assertThat(true).isFalse();
     });
   }
 
   @Test
   public void should_fail_if_one_entry_does_not_satisfy_the_given_requirements() {
```

Comment:
now that we report all the errors I think we should have two `should_fail` tests, one making sure it fails when only one entry does not satisfy the requirements (named like this test) and another that verifies we report all the errors (basically the body of that test to be renamed as `should_report_all_the_entries_not_satisfying_the_given_requirements`